src/studio-header/Icons.jsx

```diff
@@ -0,0 +1,47 @@
+// This file was copied from edx/frontend-component-header-edx.
```

Comment:
There were no modifications made to this file. It is an exact copy of the corresponding file in the `edx/frontend-component-header-edx` repository.